functions/scheduleinstance/index.js

```diff
@@ -31,37 +31,31 @@ const compute = new Compute();
  * @param {!object} callback Cloud Function PubSub callback indicating
  *  completion.
  */
-exports.startInstancePubSub = (event, context, callback) => {
+exports.startInstancePubSub = async (event, context, callback) => {
```

Comment:
I ran this updated code for the tutorial [Scheduling compute instances with Cloud Scheduler](https://cloud.google.com/scheduler/docs/start-and-stop-compute-engine-instances-on-a-schedule)

Actions:
1. Create a GCE instance `dev-instance` and a Cloud Function `startInstancePubSub`.
2. Stop the `dev-instance` instance
3. Run `startInstancePubSub` with a test trigger event targeting `dev-instance`.

Expected:
- Instance is started
- Function returns `Successfully started instance dev-instance`

Actual:
- Instance is started
- Function returns `Function failed on loading user code. Error message: File index.js or function.js that is expected to define function doesn't exist in the root directory.`

Reply:
I re-ran this code.  This time `startInstancePubSub` worked fine, but `stopInstancePubSub` did not.

Actions:
1. Create a GCE instance `dev-instance` and a Cloud Function `stopInstancePubSub`.
3. Run `stopInstancePubSub` with the test trigger event specified in the tutorial.

Expected:
- Instance is stopped
- Function returns `Successfully stopped instance dev-instance`

Actual:
- Instance is stopped
- Function returns `Error: function execution failed. Details:
Cannot read property 'Symbol(Symbol.iterator)' of undefined`

Reply:
Stack trace of the error:
`stopInstancePubSub
gwd5qb1ffhx6
TypeError: Cannot read property 'Symbol(Symbol.iterator)' of undefined at Function.all (<anonymous>) at exports.stopInstancePubSub (/srv/index.js:85:19) at <anonymous> at process._tickDomainCallback (internal/process/next_tick.js:229:7)`

Reply:
Issue resolved

- Instance is stopped
- Function returns `Successfully stopped instance dev-instance`